site/pages/provisional-fr.hbs

```diff
@@ -105,13 +105,13 @@
 		<th>CSS <code>.bg-cover</code></th>
```

Comment:
Before this table, we will need a info box that explain the deprecated "experimental" CSS and they need to be changed for "provisional"

Reply:
we already have this. Do you want something specific for css? I assume this comment was written before i added the code below. 
<div class="alert alert-info">
	<h2><code>.experimental</code> depricated</h2>
	<p><code>.experimental</code> is now deprecated and it will not be supported in 6 months.
	It has to be replaced by <code>.provisional</code></p>
</div>